storybook/src/components/PageMenu/PageMenu.stories.tsx

```diff
@@ -12,15 +12,7 @@ const meta = {
   component: PageMenu,
   args: {
     alignEnd: false,
-    wrap: true,
-  },
-  argTypes: {
-    alignEnd: {
-      control: 'boolean',
-    },
-    wrap: {
-      control: 'boolean',
-    },
+    wrap: undefined,
```

Comment:
I liked how the Args Table would show the default values if we specify them but I understand and agree we don’t want them to appear in the Code View.

Can we remove `alignEnd` with the same reasoning?

Stronger yet: why would we still define args in `meta` at all?

Reply:
I don't think we need to, `alignEnd` doesn't show in the code view. I generally think it's better when booleans have a default value. I only changed this line because `wrap` has a default value of `true`, which means it shows in the doc view.